### ISSUE TITLE
need to update doc for volmap

### ISSUE THREAD

**Issue by hainm** [Amber-MD/cpptraj#159]
should update the default values for some optional keywords. For example in cpptraj doc (v15)

`peakcut <cutoff> The minimum density required to consider a local maximum a
’density peak’ in the outputted peak file.
`

Look at the code `  peakcut_ = actionArgs.getKeyDouble("peakcut", 0.05);`

**Comment by hainm** [comment]
> The buffer is ignored if the center and size are specified (see below) (page 592)

@swails 

Should use 'either' (vs and) here?

there is no `center` in below commands and I still got 
`Error: [volmap] Not all arguments handled: [ buffer 2.0 ]`

```bash
parm data/tz2.ortho.parm7
trajin data/tz2.ortho.nc 1 1
rms first :1-13
center :1-13 mass origin
volmap output/dummy.dat 0.5 0.5 0.5 :WAT@O buffer 2.0 centermask !:1-13 radscale 1.36 
```

Additionally, if both 'center' and 'size' need to appear together, there is no checking in cpptraj yet.

**Comment by drroe** [comment]
Closed via #159
